awscli/examples/ec2/describe-vpc-classic-link-dns-support.rst

```diff
@@ -0,0 +1,22 @@
+**To describe ClassicLink DNS support for your VPCs**
+
+This example describes the ClassicLink DNS support status of all of your VPCs. 
+
+Command::
+
+  aws ec2 describe-vpc-classic-link-dns-support
+
+Output::
+
+  {
+    "Vpcs": [
+      {
+        "VpcId": "vpc-614cc409", 
```

Comment:
These look real, any reason they aren't like those in the other file (`vpc-88888888`)?